src/main/frontend/components/query_table.cljs

```diff
@@ -39,9 +39,23 @@
     (:block/name item)
     (get-in item [:block/properties sort-by-column])))
 
+(defn- locale-compare
+  "Use locale specific comparison for strings and general comparison for others."
+  [x y]
+  (if (and (string? x) (string? y))
+    (.localeCompare x y)
+    (< x y)))
+
+(defn- locale-compare-desc
+  "Use locale specific descedent comparison for strings and general comparison for others."
+  [x y]
+  (if (and (string? x) (string? y))
+    (.localeCompare y x)
+    (> x y)))
+
 (defn- sort-result [result {:keys [sort-by-column sort-desc?]}]
   (if (some? sort-by-column)
-    (let [comp (if sort-desc? > <)]
+    (let [comp (if sort-desc? locale-compare-desc locale-compare)]
```

Comment:
I'm fine with the change.
Regarding the tests, I'm not sure how to add them. BTW, because this is locale dependent, I think it's not easy to write automated tests.

Reply:
Thank you for the quick update :rocket:
There is a way to [emulate the locale in playwright](https://playwright.dev/docs/emulation#locale--timezone), if you decide to take a shot at it.